### PR TITLE
Fix encoding problem and small authorship corrections

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Something weird happened with @eduardo-rodrigues last PR that made the txt files not
be properly in utf-8. Fixed now with this PR.

Resulting authors.tex file copied to LaTeX area for roadmap.

All cosmetic fixes also made in upstream Google Doc.

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
Thanks very much for this! No idea what went wrong with my local editing. I did check the encoding in the Atom programme I was using and it showed utf-8. No idea. Glad it's fixed.

**Comment by eduardo-rodrigues** [comment]
Good to go I would say.

File: CWP/papers/roadmap/authors/address.txt [hunk 1]
```diff
@@ -85,7 +85,7 @@ ULund: Fysiska institutionen, Lunds Universitet, Lund, Sweden
 UManchester: School of Physics and Astronomy, University of Manchester, Manchester, United Kingdom
 UMaribor: University of Maribor, Ljubljana, Slovenia
 UMich: Department of Physics, The University of Michigan, Ann Arbor, MI, USA
-UMinho: Departamento de Física, Universidade do Minho, Braga, Portugal
+UMinho: Departamento de Física, Universidade do  Minho, Braga, Portugal
```
[the remark this answers is not in the document]
Comment (jouvin) [reply CWP/papers/roadmap/authors/address.txt]: May be better to keep it if you want this PR to reflect what was submitted to Archive. Next update from the Google Docs will fix it.

**Comment by graeme-a-stewart** [comment]
@jouvin - all those fixes were done, although of course latex doesn't care about duplicated white space.

I found one more real error - we had two entries for Oslo, which is now corrected. *This is not corrected in arXiv yet and I am reluctant to do it as I don't want to add any more delay to processing the article.*

Everything is in sync with the Google Doc at this point.

**Comment by graeme-a-stewart** [comment]
Thanks for spotting that @eduardo-rodrigues - corrected.

**Pull request merged**